Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/onboarding/OnboardingDeltaInteroperabilityFragment.kt

```diff
@@ -0,0 +1,45 @@
+package de.rki.coronawarnapp.ui.onboarding
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import de.rki.coronawarnapp.databinding.FragmentOnboardingDeltaInteroperabilityBinding
+import de.rki.coronawarnapp.ui.main.MainActivity
+import de.rki.coronawarnapp.ui.viewmodel.InteroperabilityViewModel
+
+class OnboardingDeltaInteroperabilityFragment : Fragment(), View.OnClickListener {
+    private var _binding: FragmentOnboardingDeltaInteroperabilityBinding? = null
+    private val binding: FragmentOnboardingDeltaInteroperabilityBinding get() = _binding!!
+
+    private val interoperabilityViewModel: InteroperabilityViewModel by activityViewModels()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentOnboardingDeltaInteroperabilityBinding.inflate(inflater)
+        binding.lifecycleOwner = this
+        binding.interopViewModel = interoperabilityViewModel
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        interoperabilityViewModel.saveInteroperabilityUsed()
+
+        binding.onboardingButtonNext.setOnClickListener(this)
+        binding.onboardingButtonBack.buttonIcon.setOnClickListener(this)
+    }
+
+    private fun navigateBack() {
+        (activity as MainActivity).goBack()
+    }
+
+    override fun onClick(view: View?) {
+        navigateBack()
```

Comment:
as discussed in https://teams.microsoft.com/l/entity/com.microsoft.teamspace.tab.wiki/tab::32332847-46cb-4995-a3eb-e183b8c68f28?context=%7B%22subEntityId%22%3A%22%7B%5C%22pageId%5C%22%3A21%2C%5C%22sectionId%5C%22%3A22%2C%5C%22origin%5C%22%3A2%7D%22%2C%22channelId%22%3A%2219%3Ac51bf302889a44e28119b0efc59b8f7d%40thread.tacv2%22%7D&tenantId=42f7676c-f455-423c-82f6-dc2d99791af7 please call a method from view model and let live data observer handle navigation